parquet-hadoop/src/main/java/org/apache/parquet/hadoop/util/HadoopPositionOutputStream.java

```diff
@@ -61,8 +62,13 @@ public void flush() throws IOException {
 
   @Override
   public void close() throws IOException {
+    if (closed) {
```

Comment:
safest to use an atomic bool here
```
if (closed.getAndSet(true) { 
  return;
}
```

streams are nominally not thread safe, but cross stream use happens. Making close() itself thread safe is low cost and avoids problems on cleanup